PythonTool/TagExpansion.py

```diff
@@ -61,70 +58,53 @@ def get_vector_tags(mb, element):
 
     # If there are none of the specified mesh elements, print a warning and exit.
     if len(element_list) == 0:
-        print("WARNING: No hex elements were found in this data file.")
+        print("WARNING: No elements of this type were found in the data file.")
```

Comment:
rather than just stating "this type" might be more useful to write out what the value of `element` is.

Reply:
This can also be done with raising an error message rather than a print statement